### PR TITLE
Update of requirements.txt (removal of versions) to enable scivision to load pretrained model

### PR DESCRIPTION
**Pull request by Tonks684**

### CODE REVIEW

File: .scivision/model.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: VEdge Detector
-url: https://github.com/MartinSJRogers/VEdge_Detector_scivision.git
+url: https://github.com/alan-turing-institute/VEdge_Detector_scivision/
 import: vedge_detector
```
Comment (IFenton) [inline .scivision/model.yml]: I think this should be https://github.com/MartinSJRogers/VEdge_Detector_scivision/

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
-tensorflow==2.2
-keras==2.3.1
-protobuf==3.20.1
+tensorflow
```
Comment (IFenton) [inline requirements.txt]: I would be inclined to keep these pinnings, but as a lower bound (e.g. tensorflow>=2.2), as the model may not work for versions below that (assuming that still solves the problem)

**Comment by Tonks684** [comment]
Changes made

**Comment by MartinSJRogers** [comment]
Thanks both, this should also hopefully resolve issues @acocac and I had in previous conversations about version conflicts.

**Pull request merged**